manifests/charts/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -1,7 +1,7 @@
 {{- define "mesh" }}
     # The trust domain corresponds to the trust root of a system.
     # Refer to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain
-    trustDomain: "cluster.local"
+    trustDomain: {{ .Values.meshConfig.trustDomain }}
```

Comment:
This should not be required. `values.meshConfig` is merged on top of the `"mesh"`.

If you try setting the value you should see it applied - it worked for me just now

Reply:
You're right. I feel stupid, I probably made other adjustments to make the trust domain work alongside this. I've now double-checked and this PR doesn't do anything on the output. I'm closing this PR. Sorry for the incorrect PR, the above in the helm charts is confusing though.